lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -723,6 +723,54 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return out.join( ',' );
 	});
 
+	/**
+		* return sliced TypedArray.
```

Comment:
Your code should look very similar to https://github.com/stdlib-js/stdlib/blob/4b1d77d2bd001d5970ce93230765a579fb41349d/lib/node_modules/%40stdlib/array/bool/lib/main.js#L1788. Currently, it does not.